apis/externalsecrets/v1/secretstore_vault_types.go

```diff
@@ -324,6 +324,11 @@ type VaultJwtAuth struct {
 // VaultCertAuth authenticates with Vault using the JWT/OIDC authentication
 // method, with the role name and token stored in a Kubernetes Secret resource.
 type VaultCertAuth struct {
+	// Path where the Certificate authentication backend is mounted
+	// in Vault, e.g: "cert"
+	// +kubebuilder:default=cert
+	Path string `json:"path"`
```

Comment:
This is a breaking change. This should be an `//+optional` setting.

Reply:
@Skarlso thanks, fixed!